### PR TITLE
[EN] Options

### PR DESCRIPTION
**Pull request by MegaApplePi**
- redo Options (English only)
- moved `Options/Univeral_Offset` to `List_of_Guides/How_to_use_the_Offset_Wizard`
- redo `List_of_Guides/How_to_use_the_Offset_Wizard`
- redo `Keyboard_Bindings` (formerly `Key_Bindings`)

### CODE REVIEW

File: wiki/List_of_Guides/How_to_use_the_Offset_Wizard/en.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Click and hold on the selector, then drag left or right.
+You can also place the cursor on the selector and move it using the left and right arrow keys.
+Continue adjusting until you find the position where the two different sounds are heard simultaneously.
+Alternatively, you can use `Offset Wizard` for a more graphical representation of ticks.
+
+Unlike Local Song Offset (specific to that difficulty only), this offset affects *every* beatmap you play. If you received an Online Offset, you can manually adjust it yourself for the difficulty you are playing using `+` or `-` to increase/decrease the beatmap's offset by 5ms (holding `Alt` while pressing `+` or `-` will allow you to change the offset by 1ms).
+
+#### Offset Wizard
+
+Click on "Run Offset Wizard" bar to adjust your Universal Audio Offset in a more graphical method.
```
Comment (Shadowtail117) [inline wiki/List_of_Guides/How_to_use_the_Offset_Wizard/en.md]: Shouldn't it be 'Click on *the* "Run Offset Wizard" bar...'?

File: wiki/List_of_Guides/How_to_use_the_Offset_Wizard/en.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+6. Click on the "Test" button.
+7. Hear the beat, see when you should be hitting the circles.
+8. Start over, close your eyes, and click to the beat. You have to click for ~16 times.
+9. Open your eyes and look how much you got your ms too early or too late.
+10. Go to options and set your Universal Offset by -X if it was too late and X if it was too early.
+11. Do 2-9 steps again, this time the too early / too late values should be smaller.
+12. Repeat 10th step until the difference gets really small (±4ms).
+13. Enjoy *actually* clicking to the rhythm!
+
+Changing the video setting to fullscreen or vice versa (full screen is recommended because of lower input lag) could make the offset require calibration. Effects volume set to 0% is recommended, because it might be delayed, therefore, could mess things up. Not sure if it works correctly for DT/NC.
```
Comment (Shadowtail117) [inline wiki/List_of_Guides/How_to_use_the_Offset_Wizard/en.md]: "...might be delayed, *and* therefore, could mess things up..." 
There's a little wiggle room on where the "and" could go, but to me this sounds the best.

File: wiki/Options/en.md [hunk 3]
```diff
@@ -11,165 +14,242 @@ To find the build of your osu!, scroll down to the lowest part of the sidebar (o
...
+| Universal offset | The offset (in milliseconds) that all beatmaps will use (in addition to the local offset). | Slider | 100%     |
+| Offset wizard    | Opens the offset wizard.                                                                   | Slider | 80%      |
+
+For more details on using the offset wizard, see [How to use Offset Wizard](/wiki/List_of_Guides/How_to_use_the_Offset_Wizard).
 
 ## Skin section
 
+![Skin icon](img/skin.jpg "Skin icon")
+
+This section is about audio related things.
```
Comment (Shadowtail117) [inline wiki/Options/en.md]: Shouldn't this be "This section is about skin related things"?

**Comment by MegaApplePi** [comment]
@Shadowtail117 this is WIP, please hold off until I am ready for reviewers

File: wiki/Options/en.md [hunk 4]
```diff
@@ -1,307 +1,443 @@
...
-    -   So, b20140323.3 means Stable Build as of 23rd of March 2014, third revision.
--   b(YYYY)(MM)(DD).(revision)(BuildType) = Test Build.
-    -   So, b20151113.8cuttingedge means Cutting Edge Build as of 13th of November 2015, eighth revision.
+When you change a setting that contained a default value, the gray vertical bar on the left will glow yellow.
+Clicking on this will reset the value back to its default value.
 
-## General section
+## General
+
+![General icon](img/general.jpg "General icon")
```
Comment (MegaApplePi) [inline wiki/Options/en.md]: alrighty ~~I took them from the old wiki anyways~~

File: wiki/Options/Keyboard_Bindings/en.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
+# Keyboard Bindings
+
+To access this dialog, open the [Options](/wiki/Options/), type in `change` then click on `Change keyboard bindings`.
+
+Here, you can change reassign keyboard keys with the listed actions.
+
+**Be aware** that the osu!client will not notify you for keys used multiple times (this includes conflicting keys)!
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: Be aware -> Note?

File: wiki/Options/Keyboard_Bindings/en.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+## osu! Standard Mode
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: I guess you can't rename the mode

File: wiki/Options/Keyboard_Bindings/en.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+| Name        | Default |
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: Default -> Default Key?

File: wiki/Options/Keyboard_Bindings/en.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+| Dash! (2x movement speed) | `Left Shift`  |
+
+## osu! Mania Mode
+
+| Name           | Default |
+|----------------|---------|
+| Increase Speed | `F3`    |
+| Decrease Speed | `F4`    |
+
+Notes:
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: bold Notes

File: wiki/Options/Keyboard_Bindings/en.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+| Name                       | Default     |
+|----------------------------|-------------|
+| Game Pause                 | `Esc`       |
+| Skip Cutscene              | `Space`     |
+| Toggle Scoreboard          | `Tab`       |
+| Increase Local Song Offset | `+` (plus)  |
+| Decrease Local Song Offset | `-` (minus) |
+| Quick Retry (hold briefly) | `~` (tilde) |
+
+Notes:
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: same

File: wiki/Options/Keyboard_Bindings/en.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+|------------------------|--------------|
+| Toggle Framerate Limit | `F7`         |
+| Toggle Chat            | `F8`         |
+| Toggle Extended Chat   | `F9`         |
+| Save Screenshot        | `F12`        |
+| Increase Volume        | `Up Arrow`   |
+| Decrease Volume        | `Down Arrow` |
+| Disable Mouse buttons  | `F10`        |
+| Boss Key               | `Insert`     |
+
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: no Notes line here?

File: wiki/Options/Keyboard_Bindings/en.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+Notes:
+
+- By default, these key binds are positioned the same way how the mod icons are positioned.
+- You may need to press the key twice to access mod icons hidden underneath them.
+  - e.g. You will need to press `S` twice to access Perfect.
+
+## Reset all Bindings
+
+Caution: this button does **not** confirm you if you had really wanted to do so!
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: bold Caution

File: wiki/Options/Keyboard_Bindings/en.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+
+This button will reset all bindings to their defaults listed above.
+
+## Trivia
+
+For mod select:
```
Comment (TPGPL) [inline wiki/Options/Keyboard_Bindings/en.md]: bold L138

File: wiki/Options/en.md [hunk 13]
```diff
@@ -1,307 +1,443 @@
 # Options
 
-Click **Options** or press **O** in the Intro Screen to access the Options sidebar and change the way osu! behaves.
+![Options menu](img/options_basic.jpg "Options menu (yes, that sidebar at left)")
```
Comment (TPGPL) [inline wiki/Options/en.md]: (yes, that sidebar at left) -> (sidebar at left), maybe?

File: wiki/Options/Keyboard_Bindings/en.md (hunk 6 above)
Comment (MegaApplePi) [inline wiki/Options/Keyboard_Bindings/en.md]: /shrug this is what the section name is called from the osu!client itself

File: wiki/Options/Keyboard_Bindings/en.md (hunk 7 above)
Comment (MegaApplePi) [inline wiki/Options/Keyboard_Bindings/en.md]: /shrug if I did that then I would need to add more spaces to the columns... `Default` seems to be self-explanatory.

**Pull request merged**